aries-site/src/examples/templates/forms/SignInExample.js

```diff
@@ -161,7 +161,6 @@ export const SignInExample = () => {
           onSubmit={({ value, touched }) => onSubmit({ value, touched })}
         >
           <FormField
-            required
```

Comment:
I think we still want the `required` prop in all of these in order for validation to run properly. However, it seems we may need to enhance grommet to allow the asterisk * to be removed in certain required cases.

Right now, grommet-theme-hpe sets `requiredIndicator` to `true` in the theme, but it seems maybe we need a prop to allow the theme definition to be overridden. Maybe adding object value to `required` such as:
```
required={{
   indicator: false
}}
```
OR
```
required="noIndicator"
```

Reply:
I can make an adjustment on grommet.. Im trying to think of the best approach I can add the prop then add guidance  around why and when this prop will be used